### PR TITLE
[XENOARCH] Xenoarch Spur and Polarstar

### PR DESCRIPTION
**Pull request by alexkar598**
also comes with spacing fix for miscellaneous.dm and code quality upgrade for NTSL,both functionally the same but stops breaking my linter. wip not done il come up with a pr body later also spoilers

### CODE REVIEW

File: yogstation/code/modules/scripting/Parser/Expressions.dm [hunk 1]
```diff
@@ -166,12 +166,12 @@
 	- <ParseParamExpression()>
 */
 		ParseExpression(list/end=list(/token/end), list/ErrChars=list("{", "}"), check_functions = 0, check_assignments = 1)
-			var/stack
-				opr=new
-				val=new
+			var/stack/opr = new
+			var/stack/val = new
+
```
Comment (Altoids1) [inline yogstation/code/modules/scripting/Parser/Expressions.dm]: Don't edit like 3 mutually exclusive modules of the game in one PR.

Make your NTSL changes and whatever else separate from this Xenobio stuff.

**Review by Altoids1 — DISMISSED** [review]

File: yogstation/code/modules/scripting/Parser/Expressions.dm (hunk 1 above)
Comment (alexkar598) [reply yogstation/code/modules/scripting/Parser/Expressions.dm]: it broke my linter so i had to fix that,do i really need to make another PR for each thing i fix thats not related to this PR but literally doesnt impact a thing in the game

**Comment by alexkar598** [comment]
@Altoids1 ready for review

**Review by Amelia0010 — APPROVED** [review]

**Comment by Amelia0010** [comment]
@alexkar598 just write up your PR body and ping-pong me

File: yogstation/code/modules/scripting/Parser/Expressions.dm (hunk 1 above)
Comment (ThatLing) [reply yogstation/code/modules/scripting/Parser/Expressions.dm]: Yes

**Review by ThatLing — DISMISSED** [review]
Modularize your PRs >:(

**Pull request merged**